src/utils/parseDependency.ts

```diff
@@ -0,0 +1,3 @@
+export const parseDependencyValue = (value: string) => {
+    return value.split(',').map((taskId: string) => taskId.trim());
+};
```

Comment:
Can you please write test for this too?

Reply:
I have already written

Reply:
already done

Reply:
Cool. Please do specify where

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/689/files#diff-6eca570aba73c02b7335bdda94c2b075dc20b9d502721d82fce42d3b5fdba539